### PR TITLE
Change from insider SDK to public 18362 SDK

### PR DESCRIPTION
**Pull request by jevansaks**

### CODE REVIEW

File: SdkVersion.props [hunk 1]
```diff
@@ -6,16 +6,17 @@
...
+	<SDKVersionInsider>10.0.18362.0</SDKVersionInsider>
   </PropertyGroup>
   <PropertyGroup>
     <UseInsiderSDK>true</UseInsiderSDK>
-    <!-- By default we use the publicly shipped SDK version which is RS5 now -->
-    <MuxSdkVersion Condition="$(UseInsiderSDK) != 'true'">$(SDKVersionRS5)</MuxSdkVersion>
+    <!-- By default we use the publicly shipped SDK version which is 19H1 now -->
+    <MuxSdkVersion Condition="$(UseInsiderSDK) != 'true'">$(SDKVersion19H1)</MuxSdkVersion>
 
     <!-- Setting UseInsiderSDK will allow the code to build to the newest RS6 SDK
```
Comment (licanhua) [inline SdkVersion.props]:
```suggestion
    <!-- Setting UseInsiderSDK will allow the code to build to the newest insider SDK 
```

File: SdkVersion.props [hunk 2]
```diff
@@ -6,16 +6,17 @@
 	<SDKVersionRS3>10.0.16299.0</SDKVersionRS3>
 	<SDKVersionRS4>10.0.17134.0</SDKVersionRS4>
 	<SDKVersionRS5>10.0.17763.0</SDKVersionRS5>
-	<SDKVersion19H1Insider>10.0.18323.0</SDKVersion19H1Insider>
+	<SDKVersion19H1>10.0.18362.0</SDKVersion19H1>
+	<SDKVersionInsider>10.0.18362.0</SDKVersionInsider>
   </PropertyGroup>
   <PropertyGroup>
     <UseInsiderSDK>true</UseInsiderSDK>
```
Comment (licanhua) [inline SdkVersion.props]: Should you remove this line? Here we set UseInsiderSDK to true. and we don't need to use conditional in MuxSdkVersion

File: docs/developer_guide.md [hunk 3]
```diff
@@ -23,15 +23,16 @@ Install latest VS2017 (15.9 or later) from here: http://visualstudio.com/downloa
...
 https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk
 
-You will also need to install The Windows 10 Insider SDK 18312. The easiest way 
+<!-- 
+You will also need to install The Windows 10 Insider SDK 18323. The easiest way 
 to install this is to run the Install-WindowsSdkISO.ps1 script from this repo in
 an Administrator Powershell window:
 
- `.\build\Install-WindowsSdkISO.ps1 18323`
+ `.\build\Install-WindowsSdkISO.ps1 18323` -->
```
Comment (licanhua) [inline docs/developer_guide.md]:
```suggestion
 `.\build\Install-WindowsSdkISO.ps1 18362` -->
```

Comment (jevansaks) [reply docs/developer_guide.md]: Note that this is commented out. It's not needed anymore but I wanted to leave it for the next person who wants to retarget to insider.

File: SdkVersion.props (hunk 2 above)
Comment (jevansaks) [reply SdkVersion.props]: We currently aren't using an insider SDK but i'm sure we will again soon.

**Comment by jevansaks** [comment]
FYI I forgot to remove all the USE_INSIDER_SDK #ifdef checks so I just pushed a commit to do that.

**Pull request merged**